web/themes/custom/pineapple/templates/content/node--article--teaser.html.twig

```diff
@@ -71,14 +71,20 @@
  */
 #}
 {{ attach_library('classy/node') }}
+{% if parent == 'content_spotlight' %}
+  {% set card_layout = 'horizontal' %}
+{% else %}
+  {% set card_layout = 'vertical' %}
+{% endif %}
 {% if content.field_a_teaser_image|field_value %}
-  {% set image_url = bamboo_render_image_style(content.field_a_teaser_image|field_target_entity.field_media_image.entity.uri.value, 'vertical_teaser') %}
+  {% set image_url = bamboo_render_image_style(content.field_a_teaser_image|field_target_entity.field_media_image.entity.uri.value, card_layout~'_teaser') %}
```

Comment:
Add space between variable, tilde and single quote also do for variable below

Reply:
done